components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesBookmarksStorageTest.kt

```diff
@@ -143,13 +144,30 @@ class PlacesBookmarksStorageTest {
             assertEquals(insertedItem, this[0].guid)
         }
 
+        with(bookmarks.getRecentBookmarks(1, TimeUnit.DAYS.toMillis(1))) {
+            assertEquals(insertedItem, this[0].guid)
+        }
+
+        with(bookmarks.getRecentBookmarks(1, 1)) {
```

Comment:
This is technically a bit flaky as it relies on execution time of the test until this point being < 1ms, which of course will likely always be true, but still.

If we wanted to improve this we could use a fake storage or use our fake clock:
https://github.com/mozilla-mobile/android-components/blob/main/components/feature/[…]a/mozilla/components/feature/search/region/RegionManagerTest.kt

https://github.com/mozilla-mobile/android-components/blob/main/components/feature/search/src/main/java/mozilla/components/feature/search/region/RegionManager.kt#L43

Not sure if worth it.

Reply:
Will look into it!